README.md

```diff
@@ -153,18 +153,25 @@ Short description.
 
   App that calculates the climate impact of your daily choices by connecting to apps and services you already use: your personal and automated carbon tracker.
 
+- [Wren](https://projectwren.com) | United States | 2019
```

Comment:
This is probably a leftover from the other pull request, can you remove it please?

Reply:
Yeah, my bad!